src/libproc_macro/bridge/client.rs

```diff
@@ -202,10 +202,15 @@ impl Clone for Literal {
     }
 }
 
-// FIXME(eddyb) `Literal` should not expose internal `Debug` impls.
 impl fmt::Debug for Literal {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-        f.write_str(&self.debug())
+        f.debug_struct("Literal")
+            .field("kind", &format_args!("{}", &self.debug_kind()))
```

Comment:
```suggestion
            .field("kind", &self.debug_kind())
```
?

Or is it used to cancel the effect of `{:#?}`, similarly to the suffix? Could you add a comment then?

Reply:
`self.debug_kind()` is a String so your suggestion prints it with quotes around, which is not what I want. The {} representation prints without quotes, as in `kind: Float`.